sqlglot/expressions.py

```diff
@@ -2314,6 +2314,16 @@ class Fetch(Expression):
     }
 
 
+class Grant(Expression):
+    arg_types = {
+        "privilages": True,
```

Comment:
There's a typo here:
```suggestion
        "privileges": True,
```